src/helpers/game_utils.ts

```diff
@@ -600,3 +603,13 @@ export function isPowerHour(): boolean {
         (powerHour) => currentHour >= powerHour && currentHour <= powerHour + 1
     );
 }
+
+/**
+ * @param artistName - the artist name
+ * @returns the normalized artist name for state entry
+ */
+export function normalizeArtistNameEntry(artistName: string): string {
+    return normalizePunctuationInName(
+        artistName.toLowerCase().replaceAll(" ", "")
```

Comment:
`normalizePunctuationInName` already removes spaces since it's a REMOVED_CHARACTER:

```ts
const REMOVED_CHARACTERS = /[\|’\ '?!.\-,:;★*´\(\)\+\u200B…]/g;
```